lib/ansible/module_utils/tacp.py

```diff
@@ -196,9 +199,21 @@ def is_valid_uuid(uuid_to_test, version=4):
 
 def api_response_to_dict(api_response):
```

Comment:
What is the primary goal of this function? To turn a response object into some unstructured dictionary? For what purpose? To print out to the console?

Reply:
primary goal is to turn a response object into a dictionary for outputting the details of a created resource into the ansible result for the creation task, with the intent of making that available for future tasks that may use the task's results.